packages/ra-ui-materialui/src/auth/Login.tsx

```diff
@@ -114,18 +123,25 @@ class Login extends Component {
 
     componentDidUpdate() {
         if (!this.backgroundImageLoaded) {
-            this.lazyLoadBackgroundImage(true);
+            this.lazyLoadBackgroundImage();
         }
     }
 
     render() {
-        const { classes, className, loginForm, ...rest } = this.props;
+        const {
+            backgroundImage,
+            classes,
+            className,
+            loginForm,
+            staticContext,
+            ...rest
```

Comment:
Some of the props that were in `sanitizeRestProps` are missing here

Reply:
Because they were unnecessary. I checked